README.md

```diff
@@ -53,6 +53,24 @@ console.log(decodedHeader);
 
 **Note:** A falsy or malformed token will throw an `InvalidTokenError` error; see below for more information on specific errors.
 
+## Polyfilling atob
+
+This library relies on `atob()`, which is a global function available on [all modern browsers as well as every supported node environment](https://developer.mozilla.org/en-US/docs/Web/API/atob#browser_compatibility).
+
+In order to use `jwt-decode` in an environment that has no access to `atob()`, ensure to provide the corresponding polyfill in your application:
+
+```js
+import "core-js/stable/atob";
+```
+
+Some environments might not work well with polyfills and require you to import the pure function and expose it yourself instead (e.g. React Native):
+
+```js
+import atob from "core-js-pure/stable/atob";
+
+global.atob = atob;
```

Comment:
Does React Native have support for `globalThis`? That might be preferential here.

Reply:
Basedomn your investigation, I am considering reverting this and only recommend using the core-js/stable/atob, or whatever polyfill works for the platform of choise. What do you think?

Reply:
Even better, I wonder if we should also just recommend base-64, which has been succesfuly used here: https://github.com/techmatters/terraso-mobile-client/pull/567/files

The benefit of base-64 is that it's created by an active and well-known person, so I am confident. It also has 0 dependencies, so even though it hasnt had any release since 2010, I do not think there is any reason to cut releases and it shouldn't mean it's a package to avoid.

Reply:
`base-64` LGFM

Reply:
I think this is a good idea.

Reply:
I updated the PR to contain both. LML what you all think.

Reply:
Updated the PR to only mention base-64. LMK what you think.

Reply:
Looks good to me. 👍

Reply:
Just to be clear, the repo on base-64 does look kinda stale. But I am confident if needed we can reach out to mathias directly, or replace the recommendation of base-64 with core-js if problems arise.

Reply:
I think this is a fine solution @frederikprijck. I am hopeful Hermes might eventually implement this feature, so we might be able to remove this in the future sometime.